pkgs/applications/editors/emacs/elisp-packages/melpa-packages.nix

```diff
@@ -146,6 +146,9 @@ let
           '';
         });
 
+        # native compilation error
+        aio = ignoreCompilationError super.aio;
```

Comment:
This comment does not bring more information.

If the error is not nix-specific, please report it to upstream with a reproducing example without nix.  Then put the issue link in the comment or add a patch.

---

non-blocing: At the end of this file, there are a large number of overrides for native compilation stuff and they are sorted line by line.  I would put this new entry there.

Reply:
> If the error is not nix-specific, please report it to upstream with a reproducing example without nix. Then put the issue link in the comment or add a patch.

good idea, done

> I would put this new entry there.

I hopefully put the file where you suggested. Please tell me if not.

Reply:
Let's wait for upstream a bit.  Maybe there is an easy patch we can add instead.

Reply:
Ok for me. The last commit was 5 years ago so I don't have much hope.

Reply:
The author is quite active.